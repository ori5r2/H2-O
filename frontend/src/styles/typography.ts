export const TYPOGRAPHY = {
  HeadEnBold26: {
    fontFamily: 'Hyundai Sans Head',
    fontSize: '26px',
    fontWeight: 700,
    lineHeight: '36px',
    letterSpacing: '-0.01em',
  },
  HeadENBold24: {
    fontFamily: 'Hyundai Sans Head',
    fontSize: '24px',
    fontWeight: 700,
    lineHeight: '32px',
    letterSpacing: '-0.01em',
  },
  HeadENBold22: {
    fontFamily: 'Hyundai Sans Head',
    fontSize: '22px',
    fontWeight: 700,
    lineHeight: '28px',
    letterSpacing: '-0.01em',
  },
  HeadENMedium20: {
    fontFamily: 'Hyundai Sans Head',
    fontSize: '20px',
    fontWeight: 500,
    lineHeight: '28px',
    letterSpacing: '-0.01em',
  },
  HeadENBold10: {
    fontFamily: 'Hyundai Sans Head',
    fontSize: '10px',
    fontWeight: 700,
    lineHeight: '12px',
    letterSpacing: '-0.01em',
  },

  TextENMedium18: {
    fontFamily: 'Hyundai Sans Text',
    fontSize: '18px',
    fontWeight: 500,
    lineHeight: '28px',
    letterSpacing: '-0.01em',
  },
  TextENMedium16: {
    fontFamily: 'Hyundai Sans Text',
    fontSize: '16px',
    fontWeight: 500,
    lineHeight: '24px',
    letterSpacing: '-0.01em',
  },
  TextENMedium14: {
    fontFamily: 'Hyundai Sans Text',
    fontSize: '14px',
    fontWeight: 500,
    lineHeight: '22px',
    letterSpacing: '-0.01em',
  },
  CaptionENMedium14: {
    fontFamily: 'Hyundai Sans Text',
    fontSize: '14px',
    fontWeight: 500,
    lineHeight: '28px',
    letterSpacing: '-0.01em',
  },
  CaptionENMedium12: {
    fontFamily: 'Hyundai Sans Text',
    fontSize: '12px',
    fontWeight: 500,
    lineHeight: '24px',
    letterSpacing: '-0.01em',
  },

  HeadKRBold32: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '32px',
    fontWeight: 700,
    lineHeight: '36px',
    letterSpacing: '-0.03em',
  },
  HeadKRBold26: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '26px',
    fontWeight: 700,
    lineHeight: '36px',
    letterSpacing: '-0.03em',
  },
  HeadKRBold24: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '24px',
    fontWeight: 700,
    lineHeight: '32px',
    letterSpacing: '-0.03em',
  },
  HeadKRBold22: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '22px',
    fontWeight: 700,
    lineHeight: '28px',
    letterSpacing: '-0.03em',
  },
  HeadKRBold20: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '20px',
    fontWeight: 700,
    lineHeight: '24px',
    letterSpacing: '-0.03em',
  },
  HeadKRBold18: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '18px',
    fontWeight: 700,
    lineHeight: '28px',
    letterSpacing: '-0.03em',
  },
  HeadKRBold12: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '12px',
    fontWeight: 700,
    lineHeight: '18px',
  },

  TextKRBold12: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '12px',
    fontWeight: 700,
    lineHeight: '18px',
    letterSpacing: '-0.03em',
  },

  HeadKRMedium26: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '26px',
    fontWeight: 500,
    lineHeight: '36px',
    letterSpacing: '-0.03em',
  },
  HeadKRMedium24: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '24px',
    fontWeight: 500,
    lineHeight: '32px',
    letterSpacing: '-0.03em',
  },
  HeadKRMedium22: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '22px',
    fontWeight: 500,
    lineHeight: '28px',
    letterSpacing: '-0.03em',
  },
  HeadKRMedium20: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '20px',
    fontWeight: 500,
    lineHeight: '24px',
    letterSpacing: '-0.03em',
  },
  HeadKRMedium18: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '18px',
    fontWeight: 500,
    lineHeight: '28px',
    letterSpacing: '-0.03em',
  },
  HeadKRMedium16: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '16px',
    fontWeight: 500,
    lineHeight: '24px',
    letterSpacing: '-0.03em',
  },
  HeadKRMedium14: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '14px',
    fontWeight: 500,
    lineHeight: '22px',
    letterSpacing: '-0.03em',
  },

  TextKRMedium18: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '18px',
    fontWeight: 500,
    lineHeight: '18px',
    letterSpacing: '-0.03em',
  },
  TextKRMedium16: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '16px',
    fontWeight: 500,
    lineHeight: '18px',
    letterSpacing: '-0.03em',
  },
  TextKRMedium14: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '14px',
    fontWeight: 500,
    lineHeight: '22px',
    letterSpacing: '-0.03em',
  },
  TextKRMedium12: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '12px',
    fontWeight: 500,
    lineHeight: '18px',
    letterSpacing: '-0.03em',
  },
  TextKRMedium10: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '10px',
    fontWeight: 500,
    lineHeight: '16px',
    letterSpacing: '-0.03em',
  },

  HeadKRRegular26: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '26px',
    fontWeight: 400,
    lineHeight: '36px',
    letterSpacing: '-0.03em',
  },
  HeadKRRegular24: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '24px',
    fontWeight: 400,
    lineHeight: '32px',
    letterSpacing: '-0.03em',
  },
  HeadKRRegular22: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '22px',
    fontWeight: 400,
    lineHeight: '28px',
    letterSpacing: '-0.03em',
  },
  HeadKRRegular20: {
    fontFamily: 'Hyundai Sans Head KR',
    fontSize: '20px',
    fontWeight: 400,
    lineHeight: '24px',
    letterSpacing: '-0.03em',
  },

  TextKRRegular18: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '18px',
    fontWeight: 400,
    lineHeight: '18px',
    letterSpacing: '-0.03em',
  },
  TextKRRegular16: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '16px',
    fontWeight: 400,
    lineHeight: '18px',
    letterSpacing: '-0.03em',
  },
  TextKRRegular14: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '14px',
    fontWeight: 400,
    lineHeight: '22px',
    letterSpacing: '-0.03em',
  },
  TextKRRegular12: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '12px',
    fontWeight: 400,
    lineHeight: '18px',
    letterSpacing: '-0.03em',
  },
  TextKRRegular10: {
    fontFamily: 'Hyundai Sans Text KR',
    fontSize: '10px',
    fontWeight: 400,
    lineHeight: '16px',
    letterSpacing: '-0.03em',
  },
} as const;

export type TypographyType = typeof TYPOGRAPHY;
export type TypographVariant = keyof TypographyType;
