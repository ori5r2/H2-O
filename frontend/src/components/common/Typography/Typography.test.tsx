import Typography from './Typography';
import { render, screen } from '@/tests/test-util';
import { COLORS } from '@/styles/colors';
import { TYPOGRAPHY } from '@/styles/typography';

describe('Typography 컴포넌트 테스트', () => {
  it('Typograhpy가 정상적으로 렌더링 된다.', () => {
    render(<Typography data-testid='typography' />);

    expect(screen.getByTestId('typography')).toBeInTheDocument();
  });

  it('Typography font를 통해서 폰트를 변경할 수 있다.', () => {
    render(<Typography data-testid='typography' color='activeBlue' />);

    expect(screen.getByTestId('typography')).toHaveStyle(`color: ${COLORS.activeBlue}`);
  });

  it('Typography color를 통해서 색상을 변경할 수 있다.', () => {
    render(<Typography data-testid='typography' font='TextKRMedium14' />);

    expect(screen.getByTestId('typography')).toHaveStyle(TYPOGRAPHY.TextKRMedium14);
  });

  describe('Typography 스냅샷 테스트', () => {
    it('HeadEnBold26', () => {
      render(<Typography data-testid='typography' font='HeadEnBold26' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadENBold24', () => {
      render(<Typography data-testid='typography' font='HeadENBold24' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadENBold22', () => {
      render(<Typography data-testid='typography' font='HeadENBold22' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadENMedium20', () => {
      render(<Typography data-testid='typography' font='HeadENMedium20' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadENBold10', () => {
      render(<Typography data-testid='typography' font='HeadENBold10' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextENMedium18', () => {
      render(<Typography data-testid='typography' font='TextENMedium18' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextENMedium16', () => {
      render(<Typography data-testid='typography' font='TextENMedium16' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextENMedium14', () => {
      render(<Typography data-testid='typography' font='TextENMedium14' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('CaptionENMedium14', () => {
      render(<Typography data-testid='typography' font='CaptionENMedium14' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('CaptionENMedium12', () => {
      render(<Typography data-testid='typography' font='CaptionENMedium12' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRBold32', () => {
      render(<Typography data-testid='typography' font='HeadKRBold32' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRBold26', () => {
      render(<Typography data-testid='typography' font='HeadKRBold26' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRBold24', () => {
      render(<Typography data-testid='typography' font='HeadKRBold24' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRBold22', () => {
      render(<Typography data-testid='typography' font='HeadKRBold22' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRBold20', () => {
      render(<Typography data-testid='typography' font='HeadKRBold20' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRBold18', () => {
      render(<Typography data-testid='typography' font='HeadKRBold18' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRBold12', () => {
      render(<Typography data-testid='typography' font='HeadKRBold12' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRMedium26', () => {
      render(<Typography data-testid='typography' font='HeadKRMedium26' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRMedium24', () => {
      render(<Typography data-testid='typography' font='HeadKRMedium24' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRMedium22', () => {
      render(<Typography data-testid='typography' font='HeadKRMedium22' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRMedium20', () => {
      render(<Typography data-testid='typography' font='HeadKRMedium20' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRMedium18', () => {
      render(<Typography data-testid='typography' font='HeadKRMedium18' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRMedium14', () => {
      render(<Typography data-testid='typography' font='HeadKRMedium14' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextKRMedium18', () => {
      render(<Typography data-testid='typography' font='TextKRMedium18' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextKRMedium16', () => {
      render(<Typography data-testid='typography' font='TextKRMedium16' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextKRMedium14', () => {
      render(<Typography data-testid='typography' font='TextKRMedium14' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextKRMedium12', () => {
      render(<Typography data-testid='typography' font='TextKRMedium12' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextKRMedium10', () => {
      render(<Typography data-testid='typography' font='TextKRMedium10' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRMedium16', () => {
      render(<Typography data-testid='typography' font='HeadKRMedium16' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRRegular26', () => {
      render(<Typography data-testid='typography' font='HeadKRRegular26' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRRegular24', () => {
      render(<Typography data-testid='typography' font='HeadKRRegular24' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRRegular22', () => {
      render(<Typography data-testid='typography' font='HeadKRRegular22' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('HeadKRRegular20', () => {
      render(<Typography data-testid='typography' font='HeadKRRegular20' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextKRRegular18', () => {
      render(<Typography data-testid='typography' font='TextKRRegular18' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextKRRegular14', () => {
      render(<Typography data-testid='typography' font='TextKRRegular14' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextKRRegular12', () => {
      render(<Typography data-testid='typography' font='TextKRRegular12' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('TextKRRegular10', () => {
      render(<Typography data-testid='typography' font='TextKRRegular10' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });

    it('DisplayText', () => {
      render(<Typography data-testid='typography' font='DisplayText' />);

      expect(screen.getByTestId('typography')).toMatchSnapshot();
    });
  });
});
