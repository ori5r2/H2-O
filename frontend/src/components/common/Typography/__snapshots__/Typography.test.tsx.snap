// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > CaptionENMedium12 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text;
  font-size: 12px;
  font-weight: 500;
  line-height: 24px;
  letter-spacing: -0.01em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > CaptionENMedium14 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text;
  font-size: 14px;
  font-weight: 500;
  line-height: 28px;
  letter-spacing: -0.01em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > DisplayText 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 146px;
  font-weight: 700;
  line-height: 1.25;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadENBold10 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head;
  font-size: 10px;
  font-weight: 700;
  line-height: 12px;
  letter-spacing: -0.01em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadENBold22 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head;
  font-size: 22px;
  font-weight: 700;
  line-height: 28px;
  letter-spacing: -0.01em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadENBold24 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head;
  font-size: 24px;
  font-weight: 700;
  line-height: 32px;
  letter-spacing: -0.01em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadENMedium20 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head;
  font-size: 20px;
  font-weight: 500;
  line-height: 28px;
  letter-spacing: -0.01em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadEnBold26 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head;
  font-size: 26px;
  font-weight: 700;
  line-height: 36px;
  letter-spacing: -0.01em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRBold12 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 12px;
  font-weight: 700;
  line-height: 18px;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRBold18 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 18px;
  font-weight: 700;
  line-height: 28px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRBold20 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 20px;
  font-weight: 700;
  line-height: 24px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRBold22 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 22px;
  font-weight: 700;
  line-height: 28px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRBold24 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 24px;
  font-weight: 700;
  line-height: 32px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRBold26 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 26px;
  font-weight: 700;
  line-height: 36px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRBold32 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 32px;
  font-weight: 700;
  line-height: 36px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRMedium14 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 14px;
  font-weight: 500;
  line-height: 22px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRMedium16 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 16px;
  font-weight: 500;
  line-height: 24px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRMedium18 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 18px;
  font-weight: 500;
  line-height: 28px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRMedium20 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 20px;
  font-weight: 500;
  line-height: 24px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRMedium22 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 22px;
  font-weight: 500;
  line-height: 28px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRMedium24 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 24px;
  font-weight: 500;
  line-height: 32px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRMedium26 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 26px;
  font-weight: 500;
  line-height: 36px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRRegular20 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 20px;
  font-weight: 400;
  line-height: 24px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRRegular22 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 22px;
  font-weight: 400;
  line-height: 28px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRRegular24 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 24px;
  font-weight: 400;
  line-height: 32px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > HeadKRRegular26 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Head KR;
  font-size: 26px;
  font-weight: 400;
  line-height: 36px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextENMedium14 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text;
  font-size: 14px;
  font-weight: 500;
  line-height: 22px;
  letter-spacing: -0.01em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextENMedium16 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text;
  font-size: 16px;
  font-weight: 500;
  line-height: 24px;
  letter-spacing: -0.01em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextENMedium18 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text;
  font-size: 18px;
  font-weight: 500;
  line-height: 28px;
  letter-spacing: -0.01em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextKRMedium10 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text KR;
  font-size: 10px;
  font-weight: 500;
  line-height: 16px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextKRMedium12 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text KR;
  font-size: 12px;
  font-weight: 500;
  line-height: 18px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextKRMedium14 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text KR;
  font-size: 14px;
  font-weight: 500;
  line-height: 22px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextKRMedium16 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text KR;
  font-size: 16px;
  font-weight: 500;
  line-height: 18px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextKRMedium18 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text KR;
  font-size: 18px;
  font-weight: 500;
  line-height: 18px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextKRRegular10 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text KR;
  font-size: 10px;
  font-weight: 400;
  line-height: 16px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextKRRegular12 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text KR;
  font-size: 12px;
  font-weight: 400;
  line-height: 18px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextKRRegular14 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text KR;
  font-size: 14px;
  font-weight: 400;
  line-height: 22px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;

exports[`Typography 컴포넌트 테스트 > Typography 스냅샷 테스트 > TextKRRegular18 1`] = `
.emotion-0 {
  font-family: Hyundai Sans Text KR;
  font-size: 18px;
  font-weight: 400;
  line-height: 18px;
  letter-spacing: -0.03em;
}

<p
  class="emotion-0"
  data-testid="typography"
/>
`;
